src/test/components/TrackCustomMarker.test.tsx

```diff
@@ -46,9 +48,13 @@ function getMarkerPixelPosition(time: number): CssPixels {
   return (time * GRAPH_WIDTH) / SAMPLE_COUNT;
 }
 
-function setup() {
+function setup(
+  values: number[] = Array(SAMPLE_COUNT)
+    .fill(0)
+    .map((_, i) => i)
```

Comment:
I think `Array.from({ length: SAMPLE_COUNT }, (_, i) => i);` is an alternative we can use these days. But no need to change this.